docs/server/quick-start/upgrade-guide.md

```diff
@@ -88,6 +88,28 @@ If you are running KurrentDB as a service, you will need to grant the `kurrent`
 
 #### Metrics name changes
 
+::: info
+The old EventStore metric names can still be used by changing the two meter names in `metricsconfig.json` to have `EventStore` prefixes:
+
+```
```

Comment:
Can we use a diff here?
![image](https://github.com/user-attachments/assets/78acb4a0-f95b-4ab3-a15a-0a91ed345483)